app/angular/conceptual/conceptual.js

```diff
@@ -317,11 +319,35 @@ const controller = function (ModelAPI, $stateParams, $rootScope, $timeout, $uibM
 		}, 100)
 	}
 
+
+
 	const registerPaperEvents = (paper) => {
+		let spacePressed = false;
+
+		paper.$document.on('keydown', (keyboardEvent) => {
+			if (keyboardEvent.code === "Space" && !event.repeat) {
+				spacePressed = true;
+			}
+			keyboardEvent.preventDefault();
+		});
+
+		paper.$document.on('keyup', (keyboardEvent) => {
+			if (keyboardEvent.code === "Space" && !keyboardEvent.repeat) {
+				spacePressed = false;
+			}
+			keyboardEvent.preventDefault();
+		});
+
 		paper.on('blank:pointerdown', function (evt, x, y) {
 			ctrl.showFeedback(false, "");
-			configs.selectionView.startSelecting(evt);
 			ctrl.onSelectElement(null);
+			configs.selectionView.cancelSelection();
```

Comment:
Should we also `cancelSelection()` when user hit ESC key?

Reply:
Good point ✅